### PR TITLE
mysql 5.6 support

### PR DESCRIPTION
**Pull request by osheroff**
There's a couple of failing tests around datetimes, and I need to figure out how to get travis to boot up either of mysql 5.5 / 5.6, but here's the basics of 5.6 support.

@vanchi-zendesk @akshayi1

### CODE REVIEW

**Comment by akshayi1** [comment]
Thanks for the super quick turnaround! I'll test this and let you know if there are any issues.

**Comment by osheroff** [comment]
cool.  do `mvn package` if you want `bin/maxwell` to work with the source distribution

File: docs/docs/quickstart.md [hunk 1]
```diff
@@ -24,9 +25,9 @@ mysql> GRANT SELECT, REPLICATION CLIENT, REPLICATION SLAVE on *.* to 'maxwell'@'
 You'll need a version 7 of a JVM.
 
 ```
-curl -sLo - https://github.com/zendesk/maxwell/releases/download/v0.8.0/maxwell-0.8.0.tar.gz \
+curl -sLo - https://github.com/zendesk/maxwell/releases/download/v0.8.1/maxwell-0.8.1.tar.gz \
        | tar zxvf -
-cd maxwell-0.8.0
+cd maxwell-0.8.1
 ```
```
Comment (vanchi-zendesk) [inline docs/docs/quickstart.md]: Guess this will be 0.8.2?

File: src/main/java/com/zendesk/maxwell/MaxwellUpdateRowsEvent.java [hunk 2]
```diff
@@ -15,6 +16,21 @@ public MaxwellUpdateRowsEvent(UpdateRowsEvent e, Table t, MaxwellFilter f) {
...
+		super(e2, table, filter);
+		UpdateRowsEvent e =  new UpdateRowsEvent(e2.getHeader());
+
+		e.setBinlogFilename(e2.getBinlogFilename());
+		e.setColumnCount(e2.getColumnCount());
+		e.setRows(e2.getRows());
+		e.setTableId(e2.getTableId());
+		e.setUsedColumnsAfter(e2.getUsedColumnsAfter());
+		e.setUsedColumnsBefore(e2.getUsedColumnsBefore());
+		e.setReserved(e2.getReserved());
```
Comment (vanchi-zendesk) [inline src/main/java/com/zendesk/maxwell/MaxwellUpdateRowsEvent.java]: There is some scope for extracting to a method here.

**Comment by vanchi-zendesk** [comment]
:+1:

**Pull request merged**